prowler/providers/kubernetes/services/core/core_service.py

```diff
@@ -19,6 +20,9 @@ def __init__(self, audit_info):
         self.__get_pods__()
         self.config_maps = {}
         self.__list_config_maps__()
+        self.nodes = []
+        self.__list_nodes__()
+        self.in_worker_node = self.__in_worker_node__()
```

Comment:
What does this mean? That you are executing it within a cluster in a pod or in the Node server? If it means the first one we have to review mapped volumes to get to the configuration files. Please review this carefully.

Reply:
If it is inside the node server you don't need to map the values, that's what I meant.

Reply:
In either case, we check if the files exist or not